### PR TITLE
Add team for timestamp codeowners

### PR DESCRIPTION
**Pull request by haydentherapper**
Signed-off-by: Hayden Blauzvern <hblauzvern@google.com>

### CODE REVIEW

**Comment by haydentherapper** [comment]
cc @cpanato

**Review by cpanato — APPROVED** [review]
lgtm
waiting on tsc approval


after that when you transfer the repo, let me know to import that in the automation

**Review by priyawadhwa — APPROVED** [review]

**Pull request merged**
